.infra/crons.ts

```diff
@@ -42,7 +42,7 @@ export const crons: Cron[] = [
   },
   {
     name: 'personalized-digest',
-    schedule: '15 0 * * *',
+    schedule: '15 * * * *',
```

Comment:
Just moved it to every hour at 15th minute for now, its gonna overlap with daily digest load as well but overall load should be smaller 🤞 

also was thinking to move it at 30th minute but as digest grows processing will just rise so they will overlap again, open to ideas

Reply:
My take is probably we want to look eventually at a different way of processing, but not urgent maybe something to chat about during gathering?

Reply:
Its more of a concern if feed service can take it because on our side we can  definitely process more messages quickly.